src/main/kotlin/org/cascadebot/cascadebot/data/objects/PlaylistType.kt

```diff
@@ -0,0 +1,5 @@
+package org.cascadebot.cascadebot.data.objects
+
+enum class PlaylistType {
```

Comment:
Why was this converted to a kotlin file?

Reply:
No reason for it, I was mainly figuring out if there was a difference.